src/Components/EditProduct/EditForm.js

```diff
@@ -0,0 +1,73 @@
+import React from "react";
+import useForm from './useForm'
+import {Col, Row, Button } from 'react-bootstrap'
+
+const EditForm = () => {
+	const { handleChange, values } = useForm();
+	return(
+		
+		<Col >
+			<div className="form-container">
+				<form className="product-form">
+				<Row>
+					<h5>Please enter the products name</h5>
+					<input
+						onChange={handleChange}
+						value={values.productName}
+						className="form-field"
+						placeholder="Product Name:"
+						name="productName" />
+				</Row>
+				<Row>
+					<h5>Add the web link to the product</h5>
+					<input
+						onChange={handleChange}
+						value={values.Link}
+						className="form-field"
+						placeholder="Link to Product:"
+						name="Link" />
+				</Row>
+				<Row>
+					<h5>Why would you like to buy this product?</h5>
+					<input
+						onChange={handleChange}
+						value={values.ReasonToBuy}
+						className="form-field"
+						placeholder="Why you want to buy this product?"
+						name="ReasonToBuy" />
+				</Row>
+				<Row>
+					<h5>How long should I wait befor I remind you about this product?</h5>
+					<input
+						onChange={handleChange}
+						value={values.RemindMe}
+						className="form-field"
+						placeholder="Remind me about this in hoe many days?"
+						name="RemindMe" />
+				</Row>
+				<Row>
+					<h5>How long should I lock the buy button?</h5>
+					<input
+						onChange={handleChange}
+						value={values.DaysToThink}
+						className="form-field"
+						placeholder="How many days to think before I can buy."
+						name="DaysToThink" />
+				</Row>
+				<Button variant="outline-dark" aligned="left aligned">
+					<i className="fas fa-times-circle"></i>
+				</Button>
+				
+				
+				
+				<Button variant="outline-dark" aligned="right aligned">
+					<i className="fas fa-check"></i>
+				</Button>

```

Comment:
The same Row\Col trick for buttons. If you put them like below (though i'm not sure it is the best way to do, i'm not an expert here) it will align as on the screenshot in the task.

> 	
	<Row>
		<Col xs={{span:1}}>
		<Button variant="outline-dark" aligned="left aligned">
			<i className="fas fa-times-circle"></i>
		</Button>
		</Col>
		<Col xs={{span:10}}>
		</Col>
		<Col xs={{span:1}}>
		<Button variant="outline-dark" aligned="left aligned">
			<i className="fas fa-check"></i>
		</Button>
		</Col>
	</Row>

Each bootsrap's row is split in 12 columns. So what we are saying here: there is a row, first column of which is for one button, the 10 columns are blank and the last column is for the second button. 
xs is screen resolution breakpoint, meaning extra-small - the small available. If we specify it, and no other breakpoints, the rule we set for xs is going to be applied for all screens (xs and up).